neuralnetwork/genome.js

```diff
@@ -25,8 +35,22 @@ class Genome {
         return innovNumber;
     };
 
-    static getDefault = (numInputs, numHiddens, numOutputs, randomWeights = false) => {
+    static assignNodeId = (innovNum) => {
+        let id;
+        if (Genome.NODE_ID_MAP[innovNum] !== undefined) {
+            id = Genome.NODE_ID_MAP[innovNum];
+        } else {
+            id = Genome.NODE_ID++;
+            Genome.NODE_ID_MAP = id;
```

Comment:
isnt "Genome.NODE_ID_MAP" initialized as a list. but here "id" is an integer so we are assigning an integer to that variable?

Reply:
yep this is a typo. just pushed a fix